src/modules/0.0.22/aws_codedeploy/entity/deploymentGroup.ts

```diff
@@ -0,0 +1,71 @@
+import { Column, Entity, JoinColumn, ManyToOne, OneToMany, PrimaryColumn } from 'typeorm';
+
+import { cloudId } from '../../../../services/cloud-id';
+import { IamRole } from '../../aws_iam/entity';
+import { CodedeployApplication } from './application';
+import { CodedeployDeployment } from './deployment';
+
+export enum DeploymentConfigType {
+  ALL_AT_ONCE = 'CodeDeployDefault.AllAtOnce',
+  HALF_AT_A_TIME = 'CodeDeployDefault.HalfAtATime',
+  ONE_AT_A_TIME = 'CodeDeployDefault.OneAtATime',
+}
+
+export enum EC2TagFilterType {
+  KEY_AND_VALUE = 'KEY_AND_VALUE',
+  KEY_ONLY = 'KEY_ONLY',
+  VALUE_ONLY = 'VALUE_ONLY',
+}
+
+@Entity()
+export class CodedeployDeploymentGroup {
+  @PrimaryColumn()
+  @cloudId
+  name: string;
+
+  @Column({
+    nullable: true,
+  })
+  id?: string;
```

Comment:
can we rename this? we have been using `id` for the autogenerated primary db id which could create confusion.